ComPact.Tests.Shared/FakePactBrokerTagMessageHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ComPact.Tests.Shared
+{
+    public class FakePactBrokerTagMessageHandler : HttpMessageHandler
```

Comment:
Couldn't this have been done by reusing the existing FakePactBrokerMessageHandler? I feel that there's some duplicate work here.

Reply:
There is a degree of cross over for sure, but other than the fact that it's a Fake HttpMessageHandler it felt like what it was doing was quite different.

I could look at merging them if you'd prefer, I think I will result in a single HttpMessageHandler which is trying to do too many things?

I was actually hoping to use Moq to create an HttpMessageHandler for the new tests, but since SendAsync() is protected rather than public that didn't work.

Anyway, let me know if you'd like me to merge these two classes.

Reply:
Yes I think that would be better. Even though we made some different implementation choices it seems to me like their intention is very similar.